routes/api-routes.js

```diff
@@ -5,7 +5,7 @@ var db = require("../models")
 module.exports = function(app){
   //Get the route for profile information for one person
   app.get("/api/members/:id", function(req, res){
-		db.Members.findOne({
+		db.Members.findAll({
 			where: {
 				id: req.params.id
 			},
```

Comment:
Is api-routes.js still in use? I think this is better under `members-api-routes.js`

Reply:
This is going to be updated in my PR thats going up in a minute